tests/test_workflows.py

```diff
@@ -42,7 +42,10 @@ def test_hs_tz_iform_contour():
 
     # A 3-parameter power function (a dependence function).
     def _power3(x, a, b, c):
-        return a + b * x**c
+        x_pos = np.where(  # this prevents 'invalid value encountered in power' warning
```

Comment:
The inline comment is very helpful for me during code review 👍 
However, I think the comment is not self-explanatory when only the new solution will be there.
It get's its meaning from the comparison to the current solutions. Should we delete the comment?
```suggestion
        x_pos = np.where(
```